common4j/src/main/com/microsoft/identity/common/java/cache/CacheRecord.java

```diff
@@ -40,12 +39,12 @@
 @Getter
 public class CacheRecord implements ICacheRecord {
 
-    @lombok.NonNull
-    private final AccountRecord mAccount;
-    private final AccessTokenRecord mAccessToken;
-    private final RefreshTokenRecord mRefreshToken;
-    private final IdTokenRecord mIdToken;
-    private final IdTokenRecord mV1IdToken;
+    @NonNull
+    AccountRecord mAccount;
+    AccessTokenRecord mAccessToken;
+    RefreshTokenRecord mRefreshToken;
+    IdTokenRecord mIdToken;
+    IdTokenRecord mV1IdToken;
```

Comment:
Why remove `private` and `final`?

Reply:
Redundant with @ Value

Reply:
Or I could remove that and add this instead.